README.md

```diff
@@ -1,2 +1,92 @@
+- CarPicsUrl : value=<string, resolvable URL> - The library fetches from this url.  Expects a JSON array of objects with "src" parameter.  Does not discriminate on endpoint – will request from any url, and attempt to make images from the return.  Allows full flexibility in delivery. 
+
+
+### All div attribute configs: 
+- id - valid value:<string, unique> - Unique identifier of the spinner div.
+- class - valid value:"CarPicsSpinner" -  Indication that a given div is in fact a CarPicsSpinner.
+- carPicsUrl – Valid Values:<string, resolvable url, returns valid json> - This is the reference to the list of images to 'spin'.  The expected format is JSON, array of objects with src fields.  src fields are expected to be resolvable image urls, and in the expected spin order.
+- numberOfConnections - valid values:<decimal numeric string> | default "4" – Number of HTTP connections to use for making image requests.
+- autospin – valid values:"true" | default "false" - Allows spinner to spin automatically, with direction and speed futher configurable.
+- autospinDirection – valid values: "left" | default "right" -  Used if it is desired to autospin in the opposite to default direction.
+- autospinSleep – valid values:<decimal numeric string> | default "100" -used in autospin to indicate how long to wait between frames (ms).
+- disableMouse – Valid values:"true" | default "false" – used to disable manual rotation, zoom. 
+- spinOnLoad – Valid values:"true" | default "false" - Used to disable autospin until all images are loaded.  If true, sets autospin true as well.  Does not prevent mouse-spin.
+
+
+### Usage Advice: 
+ - Make the div, with whatever sizing/positioning controls desired (spinner does not enforce size or aspect ratio in order to offer most flexibility and utility). 
+ - Give the div the mandatory, and whatever desired optional configurations necessary to achieve your use case. 
+ - Add the carpicsspinner script as an async script tag in the document with the above div element in it. 
+- That's all.
+- Javascript Spinner constructor exposed as well. Accessed as: CarPicsSpinners.MakeSpinner(configuration).  Takes JSON configuration argument of this format:
+```
+{
+	numberOfConnections:<number or numeric > 0  | default 4 >,
+	autospin:<string, "true" | default "false">,
+	autospinDirection:<number, 1 or -1 >,
+	spinOnLoad:<string, "true" | default "false" >,
+	divId:<string>,
+	sourceURL:<string, resolvable url to JSON resource> ,	
+	autospinSleep:<number or numeric > 0 | default 100 > ,	
+	spinSensitivity:<number or numeric >= 0 | default 5 > ,	
+	disableMouse:<string, "true" |  default "false">
+}
+```
+
+## Feature Requests:
+Feature requests / bug reports are welcome.  Please add new issues with either Feature requests or bug reports.  The general philosophy is that you should be able to use this and abuse this without issue, hence the wild inline.html setup. 

```

Comment:
"The general philosophy is that you should be able to use this and abuse this without issue, hence the wild inline.html setup. " ->
"The general philosophy is that you should be able to use and modify this without any issues."